Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -567,7 +567,7 @@ Messages::WebDriverClient::NewWindowResponse WebDriverConnection::new_window(Jso
     //    created browsing context should be in a new OS window. In all other cases the details of how the browsing
     //    context is presented to the user are implementation defined.
     // FIXME: Reuse code of window.open() instead of calling choose_a_browsing_context
-    auto [browsing_context, window_type] = m_page_client.page().top_level_browsing_context().choose_a_browsing_context("_blank"sv, true);
+    auto [browsing_context, window_type] = m_page_client.page().top_level_browsing_context().choose_a_browsing_context("_blank"sv, true, Web::HTML::ActivateTab::No);
```

Comment:
positional boolean parameters aka I looked at the function definition 30 seconds ago and already forgot what this `true` means 😢